scripts/gulp/plugins/util/getComponentInfo.ts

```diff
@@ -87,7 +87,7 @@ const getComponentInfo = (tsConfigPath: string, filepath: string, ignoredParentI
   const isChild = !isParent;
   const parentDisplayName = isParent ? null : dirname;
   // "Field" for "FormField" since it is accessed as "Form.Field" in the API
-  const subcomponentName = isParent ? null : info.displayName.replace(parentDisplayName, '');
+  const subcomponentName = isParent ? null : info.displayName.replace(parentDisplayName!, '');
```

Comment:
Didn't got why the `!` added

Reply:
It's a non-null assertion operator (needed here when `strictNullChecks` is enabled). `parentDisplayName` will never actually be null in this case, but TS is unable to infer that for some reason.

Reply:
Alright, makes sense. Thanks :)